src/libraries/Common/src/Interop/Unix/System.Net.Security.Native/Interop.GssApiException.cs

```diff
@@ -71,13 +71,19 @@ private static string GetGssApiDisplayStatus(Status majorStatus, Status minorSta
 
             private static string? GetGssApiDisplayStatus(Status status, bool isMinor)
             {
+                if (!System.Net.NegotiateAuthenticationPal.HasSystemNetSecurityNative)
+                {
+                    // avoid calling into libSystem.Net.Security.Native.
+                    return null;
+                }
```

Comment:
Do we need this change now that `supportKerberos: HasSystemNetSecurityNative` is used? I don't think this exception is thrown anywhere outside of that code.

Reply:
It is thrown when we pass unknown package name

https://github.com/dotnet/runtime/blob/6b21cb12b5177d56053bb30073c326dba268e494/src/libraries/System.Net.Security/src/System/Net/NegotiateAuthenticationPal.Unix.cs#L458-L477

which causes this test to fail

https://github.com/dotnet/runtime/blob/d79ebf5499583d4280d9a65ef2e83b89862bd0cc/src/libraries/System.Net.Security/tests/UnitTests/NegotiateAuthenticationTests.cs#L67-L75